tests/jobs/test_scheduler_job.py

```diff
@@ -516,12 +516,12 @@ def test_process_task_instances_with_task_concurrency(
                 ti.start_date = start_date
                 ti.end_date = end_date
 
-        ti_to_schedule = []
+        ti_to_schedule = Mock()
         scheduler_job._process_task_instances(dag, task_instances_list=ti_to_schedule)
 
-        assert ti_to_schedule == [
+        ti_to_schedule.append.assert_called_once_with(
             (dag.dag_id, dag_task1.task_id, DEFAULT_DATE, TRY_NUMBER),
-        ]
+        )
```

Comment:
We don't "have to" make change for this line. Only for consistent testing style/method.